src/dmd/common/int128.d

```diff
@@ -0,0 +1,749 @@
+/* 128 bit integer arithmetic.
+ *
+ * Not optimized for speed.
+ *
+ * Copyright: Copyright D Language Foundation 2022.
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:   Walter Bright
+ * Source:    $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/common/int128.d, root/_int128.d)
+ * Documentation: https://dlang.org/phobos/dmd_common_int128.html
+ * Coverage:    https://codecov.io/gh/dlang/dmd/src/master/src/dmd/common/int128.d
+ */
+
+module dmd.common.int128;
+
+nothrow:
+@safe:
+@nogc:
+
+alias I = long;
+alias U = ulong;
```

Comment:
Shouldn't this be the size of target types?

Reply:
Nope.

Reply:
Ah, right, makes sense.

Reply:
Not necessary with D. One of the advantages of D.